src/Common/Utility.cpp

```diff
@@ -42,56 +42,6 @@
 namespace usbguard
 {
 
-  void daemonize()
```

Comment:
Please remove also the declarations: https://github.com/dkopecek/usbguard/blob/master/src/Common/Utility.hpp#L54